packages/components/ng-ui-router-breadcrumb/src/ng-ui-router-breadcrumb.service.js

```diff
@@ -12,51 +12,59 @@ export default class BreadcrumbService {
   }
 
   buildBreadcrumb(transition) {
-    return transition.promise.then(() => {
-      this.breadcrumb = [];
+    let state = this.$state.$current;
+    return transition.promise
+      .then(() => {
+        const hideBreadcrumbResolvable = state.resolvables.find(
+          (resolvable) => resolvable.token === 'hideBreadcrumb',
```

Comment:
It could be nice to update the README.md file (Usage part) regarding this new feature.